src/test/java/com/softserve/kickscooter/kickscootervehicle/management/controller/ManagementEndToEndTest.java

```diff
@@ -0,0 +1,78 @@
+package com.softserve.kickscooter.kickscootervehicle.management.controller;
+
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterCreateDto;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterInfoDto;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.client.AutoConfigureWebClient;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@RunWith(SpringRunner.class)
+@AutoConfigureWebClient(registerRestTemplate = true)
+public class ManagementEndToEndTest {
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @Test
+    void getScootersId() {
+        ResponseEntity<Iterable<UUID>> response = restTemplate.exchange( "/scooter/id",
+                HttpMethod.GET, null, new ParameterizedTypeReference<>(){});
+        System.out.println(response.getBody());
+        assertNotNull(response.getBody());
+    }
+
+    @Test
+    void getAllScooterInfo() {
+        ResponseEntity<Iterable<ScooterInfoDto>> response = restTemplate.exchange("/scooter/info-all",
+                HttpMethod.GET, null, new ParameterizedTypeReference<>(){});
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+    }
+
+
+    @Test
+    void getScooterInfo_NonExist() {
+        String url = String.format( "/scooter/info/%s", UUID.randomUUID());
+        ResponseEntity<ScooterInfoDto> response = restTemplate.getForEntity(url, ScooterInfoDto.class);
+        assertEquals( HttpStatus.NOT_FOUND, response.getStatusCode());
+        //assertNull(response.getBody());
+    }
+
+    @Ignore //test works,but incorrect deserializing UUID inside
```

Comment:
don't commit new tests with `@Ignore` annotation. It's for tests that passed before but started to fail after some changes